### PR TITLE
Feat/PrepareStreamTable

### PR DESCRIPTION
**Pull request by lianxmfor**
this PR does:
1. online store add method `PrepareStreamStore`
2. pg/mysql/sqlite/cassandra/dynamodb implement method `PrepareStore`
3. create stream feature table after create stream group
4. add column to stream feature table after create stream feature

TODO
add unit test

relate #801

### CODE REVIEW

File: internal/database/online/store.go [hunk 1]
```diff
@@ -16,8 +17,12 @@ type Store interface {
 	Import(ctx context.Context, opt ImportOpt) error
 
 	// Push streaming feature to online store
+	// Note: Make sure that the table corresponding to the stream feature already exists before executing this method
 	Push(ctx context.Context, opt PushOpt) error
 
+	// Create the stream table for the stream group
+	CreateStreamTable(ctx context.Context, entity *types.Entity, groupID int) error
```
Comment (jinghancc) [inline internal/database/online/store.go:24]: if user is to add a new feature to the existing group, should we also use this function to add a new column to the existing table?

Comment (lianxmfor) [reply internal/database/online/store.go:24]: I won't be using this function, I plan to use a new function `StreamTableAddColumn` to do this

Comment (jinghancc) [reply internal/database/online/store.go:24]: hmm maybe combine those two functions? name it as `PrepareTable`?

Comment (lianxmfor) [reply internal/database/online/store.go:24]: good idea! maybe `PrepareStreamTable` is more suitable?

Comment (jinghancc) [reply internal/database/online/store.go:24]: sounds good!

Comment (lianxmfor) [reply internal/database/online/store.go]: done

File: internal/database/online/sqlite/store.go [hunk 2]
```diff
@@ -47,3 +48,26 @@ func (db *DB) Purge(ctx context.Context, revisionID int) error {
 func (db *DB) Push(ctx context.Context, opt online.PushOpt) error {
 	panic("Implement me!")
 }
+
+func (db *DB) PrepareStreamTable(ctx context.Context, opt online.PrepareStreamTableOpt) error {
```
Comment (jinghancc) [inline internal/database/online/sqlite/store.go:52]: implementations seems pretty similar for mysql, postgres, sqlite, can you put it to `sqlutil`?

**Review by jinghancc — APPROVED** [review]
LGTM

**Review by wfxr — APPROVED** [review]
LGTM

File: internal/database/online/sqlite/store.go (hunk 2 above)
Comment (lianxmfor) [reply internal/database/online/sqlite/store.go:51]: done

**Pull request merged**